BenchmarkVCFs/BenchmarkVCFs.wdl

```diff
@@ -0,0 +1,1336 @@
+version 1.0
+
```

Comment:
since this is public, let's stick a blurb about what this does.

Reply:
added as part of a meta section